content/blog/joining-eclipse.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "Joining the Eclipse Foundation"
+description = ""
+date = 2025-02-05T00:09:00+02:00
+draft = false
+template = "blog/page.html"
+
+[taxonomies]
+authors = ["geal"]
+
+[extra]
+lead = "the Biscuit project is joining the Eclipse Foundation"
++++
+
+The Biscuit project is designed to be the core authorization component in large-scale distributed systems. As such, it needs to be stable, safe, but it also became apparent that it should provide assurances for users that it will be supported for a long time, with a clear governance.
+To that end, Biscuit is joining the [Eclipse Foundation](https://www.eclipse.org/org/), which will provide it with a stable home, support and resources to help it serve a larger number of users. We thank [Outscale](https://outscale.com/) for sponsoring the transition and walking us trough the process.
+
+Practically, this means that the project will be renamed to [Eclipse Biscuit](https://projects.eclipse.org/projects/technology.biscuit), with the same maintainers as before, Geoffroy Couprie and Clément Delafargue. The repositories will gradually move to the [Eclipse Biscuit Github organization](https://github.com/eclipse-biscuit). The specification effort after [version 3.3](/blog/biscuit-3-3/) will happen under an Eclipse working group.
+Biscuit stays a community project, open to any outside contributions, though it will now ask of contributors to sign the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/eca/), which contains a developer certificate of origin. This ensures that contributors have the right to submit their proposals, but they still own their contributions.
```

Comment:
```suggestion
Biscuit stays a community project, open to any outside contributions, though it will now ask of contributors to sign the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/eca/), which contains a developer certificate of origin. This ensures that contributors have the right to submit their proposals, but they still own their contributions.

To be clear, the Eclipse Contributor Agreement is **not** a [Contributor License Agreement (CLA)](https://en.wikipedia.org/wiki/Contributor_License_Agreement). See the [ECA FAQ](https://www.eclipse.org/legal/eca/faq/) for more information.
```